bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -111,10 +111,13 @@ spec:
           fi
           
           OVN_NODES_ARRAY=({{.OVN_NODES}})
-          if [[ "$K8S_NODE" == "${OVN_NODES_ARRAY[0]}" ]]; then
+          MASTER_NODE=$(getent ahostsv4 "${OVN_NODES_ARRAY[0]}" | grep RAW | awk '{print $1}'
+          LOCALHOST=$(getent ahostsv4 "${K8S_NODE}" | grep RAW | awk '{print $1}')
```

Comment:
`ahosts`, not `ahostsv4`. At this point everyone needs to be keeping future IPv6 compatibility in mind with all new code.

Reply:
done